bin/emacstask

```diff
@@ -6,4 +6,10 @@ if [ -z "$EA_EDITOR" ]; then
     EA_EDITOR="emacsclient -a '' -c"
 fi
 
-$EA_EDITOR -e "(progn $ELISP)"
+$EA_EDITOR -e "(progn
+                 (defvar ea-app-name \"\")
+                 (setq ea-app-name \"$EA_APP_NAME\")
```

Comment:
No need to setq here, you can put it directly into `defvar`,  For extra cool points we can use `defconst` to mark it a constant, which is pretty reasonable given that we will have the app/window name constant during the "ea popup" session.

Reply:
Thank you! 

I like the term "popup" now that you mention it. Maybe I should call the hook `ea-popup-hook`.